tests/hilti/optimization/unimplemented_hook.hlt

```diff
@@ -0,0 +1,28 @@
+# @TEST-EXEC: hiltic -d -j -D global-optimizer %INPUT > log 2>&1
+# @TEST-EXEC: btest-diff log
+
+# @TEST-EXEC: ${HILTIC} -d -j -g %INPUT -D ast-codegen >ast.0 2>&1
+# @TEST-EXEC: ${HILTIC} -d -j    %INPUT -D ast-codegen >ast.1 2>&1
+# @TEST-EXEC: /usr/bin/env diff -u ast.0 ast.1 | tail -n+3 >output.diff
```

Comment:
This `diff` usage doesn't look very portable. I think it'd be better to determine the right `diff` in `btest.cfg`
and pass it in through an environment variable.

Also, let's document what diff we need (GNU or BSD?) and check that we got it.

Reply:
I replaced this with a custom diff util which I added to `tests/Scripts`. This also allows me to roll back the changes to the Alpine Docker image.